core/src/main/java/com/amplifyframework/datastore/DataStoreItemChange.java

```diff
@@ -259,9 +263,14 @@ public String toString() {
      */
     public enum Type {
         /**
-         * An item has been saved into the DataStore.
+         * An item with a new ID has been saved into the DataStore.
+         */
+        CREATE,
```

Comment:
Why do we have what looks like the same enum here as well as in StorageItemChange.java in the DataStore project?

Reply:
True, yea. The local storage module and the public DataStore API share a number of features. In this case, the enum is entirely the same.

I could use just one, but I'd need a name for it that made sense for both.

Logically speaking, one is talking about changes to the DataStore, and one is talking about changes to an internal system component (the storage adapter). Which theoretically could be different, but in practice, aren't really.

Reply:
Ah, ok yeah if they have different meanings than that makes sense to have the two.